<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <include file="$(find velodyne_pointcloud)/launch/VLP16_points.launch" />
    <node pkg="lidar_object_detector" type="traffic_cone" name="traffic_cone" output="screen"/>
</launch>